pkg/xray/policies.go

```diff
@@ -80,13 +80,13 @@ func expandPolicy(d *schema.ResourceData) (*Policy, error) {
 	if v, ok := d.GetOk("author"); ok {
 		policy.Author = StringPtr(v.(string))
 	}
-	policyRules, err := expandRules(d.Get("rules").([]interface{}), policy.Type)
+	policyRules, err := unpackRules(d.Get("rules").([]interface{}), policy.Type)
 	policy.Rules = &policyRules
 
 	return policy, err
 }
 
-func expandRules(configured []interface{}, policyType *string) (policyRules []PolicyRule, err error) {
+func unpackRules(configured []interface{}, policyType *string) (policyRules []PolicyRule, err error) {
```

Comment:
Why the `*string`? not at all necessary

Reply:
same here. Pointers were not removed from policies.go